opengrok-web/src/test/java/org/opengrok/web/api/v1/controller/SuggesterControllerTest.java

```diff
@@ -237,6 +237,8 @@ public void testGetSuggestionUnknownField() {
         assertEquals(Response.Status.BAD_REQUEST.getStatusCode(), r.getStatus());
     }
 
+    // temporarily disabled, see https://github.com/oracle/opengrok/issues/2030
+    /*
```

Comment:
Could you use @Ignore or other means provided be junit to skip the test rather then commenting it out?